### ISSUE TITLE
Gradient overflow

### ISSUE THREAD

**Issue by patriciapampanelli** [NVIDIA/NeMo#2387]
**Describe the bug**

I'm receiving these warnings and errors when I'm training Quartznet using spanish dataset at A100:

***Warnings:***
- Could not import torchaudio. Some features might not work.  warnings.warn('Could not import torchaudio. Some features might not work.')libibverbs: 
- Warning: no userspace device-specific driver found for /sys/class/infiniband_verbs/uverbs0

***Error***:
- Gradient overflow.  Skipping step, loss scaler 0 reducing loss scale to 1.0

**Environment overview (please complete the following information)**

 - Docker image: https://ngc.nvidia.com/catalog/containers/nvidia:nemo
 - Using Enroot to start the instance
 - Following the steps of: Spanish__Cross_Language.ipynb
 - NeMo Parameters:
     - nproc_per_node=8
     - batch_size=32
     - amp_opt_level = 'O1'
 - Dataset: Mozilla Common Voice - Spanish Version 6.1
 - Running at a DGX A100 80G

**Comment by titu1994** [comment]
From the description of the error, it seems a very old version of Nemo is being used. There were breaking changes between 0.11 and 1.0.0RC1 so it would be advised to update the code to newer Nemo.

**Comment by okuchaiev** [comment]
please use 1.0.2 version (you can use nvcr.io/nvidia/nemo:1.0.1 container).
